src/hacbs/components/PipelineRunDetailsView/tabs/PipelineRunEventsTab.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  EmptyState,
+  EmptyStateIcon,
+  Title,
+  EmptyStateBody,
+  Button,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
+import { OutlinedFileImageIcon } from '@patternfly/react-icons/dist/esm/icons/outlined-file-image-icon';
+
+type PipelineRunEventsTabProps = {
+  applicationName: string;
+};
```

Comment:
You likely want to pass in the pipeline run object to the tabs and not the application name.

Reply:
yeah, neither are used because they are just placeholders for now but i changed them anyway